project-base/tests/FrontendApiBundle/Functional/Product/ProductsSearch.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\FrontendApiBundle\Functional\Product;
+
+use App\DataFixtures\Demo\BrandDataFixture;
+use App\DataFixtures\Demo\CategoryDataFixture;
+
+class ProductsSearch extends ProductsGraphQlTestCase
+{
+    public function testSearchInAllProducts(): void
+    {
+        $query = '
+            query {
+                products (first: 5, search: "book") {
+                    edges {
+                        node {
+                            name
+                        }
+                    }
+                }
+            }
+        ';
+
+        $firstDomainLocale = $this->getFirstDomainLocale();
+
+        $productsExpected = [
+            ['name' => t('Book scoring system and traffic regulations', [], 'dataFixtures', $firstDomainLocale)],
+            ['name' => t('Book of traditional Czech fairy tales', [], 'dataFixtures', $firstDomainLocale)],
+            ['name' => t('Book Computer for Dummies Digital Photography II', [], 'dataFixtures', $firstDomainLocale)],
+            ['name' => t(
+                'Book of procedures for dealing with traffic accidents',
+                [],
+                'dataFixtures',
+                $firstDomainLocale
+            )],
```

Comment:
it does not seem like this array item has to be split into multiple lines anymore... 
(and my only argument for the change is "I don't like it" 😄 )

Reply:
This was one liner, but our automatic fixers changed that

Reply:
I see